packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -274,6 +279,21 @@ export class GcdsSelect {
     }
 
     this.inheritedAttributes = inheritAttributes(this.el, this.shadowElement);
+
+    if (this.el.children) {
+      this.options = Array.from(this.el.children);
```

Comment:
Could we specifically target the option elements and be more concise by doing something like this?
`const options = Array.from(this.el.querySelectorAll('option'));`

Reply:
I used `this.el.children` to just not focus on `options` but the rest of my code just assumed the passed elements were `<options>` so it doesn't make sense to use `this.el.children`. But I just remembered `<optgroup>` are a thing so I am going to refactor the rendering to allow them to work with `<optgroup>`'s

Reply:
Gotcha, that makes sense - thank you!